### ISSUE TITLE
2nd Deposit DB Trigger (Oberammergau)

### ISSUE THREAD

**Issue by ronenreinheimer** [206Tours/DB#673]
Hi Brad,

We are taking deposits for special tours in 2020 ($400). Milanka would like you to give the DB the ability to send clients a 2nd deposit request (in 2018). Can you help us with this?

www.206tours.com/oberammergau

thank you.

**Comment by bradatkins** [comment]
Yes, this is something we can do. I've marked it as an Enhancement that we can review in the future

**Comment by ronenreinheimer** [comment]
Great!

Thank you.

We Wish you a Blessed Christmas and a Happy New Year!

________________________________
[http://www.206tours.com/sigs/30th-logo2.png]<http://www.206tours.com/>

Ronen Reinheimer
Art Director
206Tours.com<http://www.206tours.com/>
